cfgov/unprocessed/css/molecules/notification.less

```diff
@@ -0,0 +1,101 @@
+/* ==========================================================================
+   Capital Framework
+   Notification Styling
+   ========================================================================== */
+
+
+/* topdoc
+  name: Notifications
+  family: cf-molecules
+  notes:
+    - "Markup and styling for notifications"
+  patterns:
+    - name: Default Usage
+      markup: |
+        <div class="m-notification m-notification__error">
+          <span class="cf-notification_icon
+                       cf-notification_icon__error
+                       cf-icon
+                       cf-icon-error-round"></span>
+          <div class="m-notification_message">
+              This is a default alert.
+          </div>
+          <div class="m-notification_explanation">
+              This is a default explanation with a <a href="#">link</a>.
+          </div>
+        </div>
+  tags:
+    - cf-molecules
+*/
+
+@m-notification-padding__px: 15px;
+
+.m-notification {
+    /* display: none; TODO: uncomment this line when prototype is merged into filters. x*/
+    position: relative;
+
+    padding: @m-notification-padding__px;
+    padding-left: 40px;
+    margin-bottom: @m-notification-padding__px + 4px;
+
+    background: @gray-20;
+    border: 1px solid @gray;
+
+    &__success {
+        background: @green-tint;
+        border-color: @input-success;
+    }
+
+    &__warning {
+        background: @gold-20;
+        border-color: @input-warning;
+    }
+
+    &__error {
+        background: @redorange-20;
+        border-color: @input-error;
+    }
+
+    &_icon {
+        position: absolute;
+        top: @m-notification-padding__px + 2px;
+        left: @m-notification-padding__px;
+
+        color: @gray-20;
+        font-size: unit( 18px / @base-font-size-px, em );
+
+        &__success {
+            color: @green;
+        }
+
+        &__warning {
+            color: @gold;
+        }
+
+        &__error {
+            color: @redorange;
+        }
+    }
+
+    &_message {
+        .h4();
+        margin: 0;
+    }
+
+    &_explanation {
+        .h4();
+        display: block;
+        margin: 0;
+        margin-top: unit( (@grid_gutter-width / 2) / @base-font-size-px, em);
+        font-size: @base-font-size-px;
```

Comment:
Edit added in https://github.com/cfpb/cfgov-refresh/commit/3a0bec0089e84b50c83c9ac9fe0da8f46e6c85b2